ethexe/contracts/src/Mirror.sol

```diff
@@ -1,14 +1,18 @@
 // SPDX-License-Identifier: UNLICENSED
 pragma solidity ^0.8.28;
 
+import {Memory} from "frost-secp256k1-evm/utils/Memory.sol";
 import {ERC1967Utils} from "@openzeppelin/contracts/proxy/ERC1967/ERC1967Utils.sol";
 import {StorageSlot} from "@openzeppelin/contracts/utils/StorageSlot.sol";
+import {ICallbacks} from "./ICallbacks.sol";
 import {IMirror} from "./IMirror.sol";
 import {IRouter} from "./IRouter.sol";
 import {IWrappedVara} from "./IWrappedVara.sol";
 import {Gear} from "./libraries/Gear.sol";
 
 contract Mirror is IMirror {
+    address internal constant ETH_EVENT_ADDR = 0xFFfFfFffFFfffFFfFFfFFFFFffFFFffffFfFFFfF;
```

Comment:
Whould be very nice if you explain strange constants here and in other places (comments).

Reply:
this is internal sails address
https://github.com/gear-tech/sails/blob/master/rs/src/solidity.rs

Reply:
Could you specify this in comments please?